libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -440,10 +475,16 @@ void lua_scripts::repl_cleanup (void) {
             AP::FS().unlink(REPL_DIRECTORY);
         }
     }
+    // Reset memory counters
+    allocated = 0;
+    deallocated = 0;
 }
 
 void lua_scripts::run(void) {
+    hal.scheduler->delay(10000);
```

Comment:
Why do we need a delay?

Reply:
Just to make sure reviewers are paying attention ;). I was testing on real hardware and wanted to see the panics I had before getting all the memory sums to add up.